### DISCUSSION TITLE
Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by pearldrum** [cpvalente/ontime#1226] [Q&A]
Is there a keyboard shortcut to hit next in the editor, using the keyboard instead of the mouse? For example, the space bar.

**Comment by cpvalente** [comment]
Hi, I apologise forgetting to reply.

The shortcuts available only affect the rundown and navigation. Ontime provides no shortcuts that affect the runtime.
This was a decision to prevent unwanted actions which may be show critical: ie, what would happen if you pressed `space` trying to edit a field, and instead started the next event along with all its automations?

If you are looking for shortcut keys we would recommend looking into companion, the ontime module is comprehensive
Hope this helps

**Reply by pearldrum** [reply]
Thank you for your answer!
Is it possible to use companion if ontime is running on a public server?

**Reply by cpvalente** [reply]
I am unsure I understood the question correctly.
Companion needs to have access to the network where Ontime is running, regardless of whether it is a open or private network
